bin/dry-run.js

```diff
@@ -28,7 +31,22 @@ if (!measurementCids.length) {
   process.exit(1)
 }
 
+const cacheDir = fileURLToPath(new URL('../.cache', import.meta.url))
```

Comment:
FWIW, this directory is already on the `.gitignore` list.

https://github.com/filecoin-station/spark-evaluate/blob/ddfe10abd91eb440f1e5f0608b6778a737f15b81/.gitignore#L105